scripts/setup-centos8.sh

```diff
@@ -201,6 +207,8 @@ function install_velox_deps {
   run_and_time install_mvfst
   run_and_time install_fbthrift
   run_and_time install_duckdb
+  run_and_time install_cuda 11.8
+  run_and_time install_cuda 12.4
```

Comment:
I think installing cuda  (and especially  both versions) should be opt in as it's not a core dependency and a rather large one too. (it adds ~ 1G to the centos image, I don't think that will make a huge difference in CI but we might want to use a separate image for that)